section3/reflection.md

```diff
@@ -1,13 +1,36 @@
 ## Section 3 Reflection
 
 1. In Section 3, we removed even more supports and pushed you to learn a bit more independently, read through technical documentation, and use your resources more. How did that feel?
+  * I felt like I did well. 3.8/5. I had to rely on notes and going back to previous lessons
+to find information. However this was easy to locate and understand due to the notes
+I have taken.
+
+  * At this time I appreciate having some of the supports removed. It is forcing me to be
+self reliant, and commit some of these skills to memory or build a system that I can
+find the information that I need.
 
 2. Think of a web or mobile application you use regularly. Where is an example of a place that the developers probably use Arrays? Explain.
+  * The application is called Easy Army. It helps build army lists for a game. There are hundreds of different options for each nation to take. This information is most likely stored in arrays. This is so that they do not have to individually log each different option and have
+an individual call name for each.
 
 3. In your own words, explain what an Array method is.
-
+  * An array method preforms a function on an array.
 4. What is the purpose of a loop? When we would use a loop in conjunction with array vs. without?
+  * A loop is used to repeat a function without having to run a command each time. The
+  loop will continue to run until it not longer produces a true statement.
+  * We would use a loop in conjunction with an array when we run the same code for items
+  the array. We can use a loop without an array if we want something done repeatedly.
+  This could be a certain amount of time or infinite.
 
 5. If you had to teach a 5 year old about loops, how would you explain it?
-
+  * If I had to teach a 5 year old about loops I would start explaining it by using a
+  circle. Even better maybe a revolving door. You can get in the door and go around and
+  around as many times as you like until someone tells you to stop. A loop allows something
+  to be done repeatedly.
 6. What questions do you still have about Arrays and loops?
+  * At this point the power potential seems enormous for loops and arrays and when I
```

Comment:
100%, yes yes yes. Each part of the code plays a little role, all building into a giant codebase.